RR.App.Tests/Controllers/HRIS/EmployeeEvaluationTemplateItemControllerUnitTests.cs

```diff
@@ -15,8 +16,8 @@ public async Task GetAllEmployeeEvaluationTemplateItemsValidSectionReturnsOkResu
         var section = "Example Test Section";
         var expectedTemplateItems = new List<EmployeeEvaluationTemplateItemDto>
         {
-            new(1, new EmployeeEvaluationTemplateDto{Id = 101, Description = "Template 1" }, section, "Question 1"),
-            new(2, new EmployeeEvaluationTemplateDto{ Id = 102, Description = "Template 2" }, section, "Question 2")
+            new EmployeeEvaluationTemplateItemDto{Id = 1, Template = new EmployeeEvaluationTemplateDto{Id = 101, Description = "Template 1" }, Section = "Example Test Section", Question = "Question 1" },
```

Comment:
Please create an EmployeeEvaluationTemplateItemTestData.cs file and move this to it. Replace everywhere in the unit test file where you're creating a new EmployeeEvaluationTemplateItem object with the one from the TestData file. Have a look at existing examples of how the TestData files are used